api/preprints/serializers.py

```diff
@@ -227,20 +235,17 @@ class PreprintCreateSerializer(PreprintSerializer):
     id = IDField(source='_id', required=False, allow_null=True)
 
     def create(self, validated_data):
-        node = validated_data.pop('node', None)
-        if not node:
-            raise exceptions.NotFound('Unable to find Node with specified id.')
-        elif node.is_deleted:
+        node = validated_data.pop('node', {})
+        if isinstance(node, dict):
+            node = Node.objects.create(creator=self.context['request'].user, **node)
```

Comment:
This results in a blank title in the default case. Is there any way to avoid that?

Reply:
I think it's ok because the node is later updated in the preprint update method [here](https://github.com/CenterForOpenScience/osf.io/pull/7648/files#diff-2d550f722c7b3838d9064d9273e9dba9R177) -- including the title passed by the API POST calls to node in the frontend. Unless I'm misunderstanding what you mean by the default case.